src/openxr/OpenXRConfig.cpp

```diff
@@ -45,6 +51,44 @@ bool OpenXRConfig::keep_3d_linear() const {
 	}
 }
 
+int OpenXRConfig::get_view_config_type() const {
+	if (openxr_api == NULL) {
+		return 1;
+	} else {
+		switch (openxr_api->get_view_configuration_type()) {
+			case XR_VIEW_CONFIGURATION_TYPE_PRIMARY_MONO: {
+				return 0;
+			}; break;
+			case XR_VIEW_CONFIGURATION_TYPE_PRIMARY_STEREO: {
+				return 1;
+			}; break;
+			default: {
+				// we don't support the others (yet)
+				return 1;
+			}; break;
+		}
+	}
```

Comment:
Do we have an enum that defines the configuration types? I thought they were matching the `ARVRInterface::Capabilities` enum but they're off by 1.

Reply:
The `ARVRInterface::Capabilities` are flags that communicate to the user what possibilities the `XRInterface` has. i.e. in this case just to communicate that stereo support is available (not enforced).

The reasons I'm using a switch here are:
- enums in Godot start with 0 so our dropdown in the UI works properly, for some reason OpenXR starts at 1
- `XR_VIEW_CONFIGURATION_TYPE_PRIMARY_QUAD_VARJO` which we currently do not support has value `1000037000`, if we are going to support that some day I'd like that to be `2`. Same deal with `XR_VIEW_CONFIGURATION_TYPE_SECONDARY_MONO_FIRST_PERSON_OBSERVER_MSFT` which if we decide to support it would likely becomes `3`.

Reply:
Here to I added commented out placeholder code to make it clearer what the future possibilities are and how the code will change in Godot 4 which now has the foundation for supporting multiple views. 

For Varjo we still need to implement quad rendering, for observer we'll need to add support for classifying different usages for different viewports and defining an extra camera for this purpose. Those are all future features to look at.